### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by nachosala89**
Adapted all sections to the desktop version.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @nachosala89, 

**This is a great project you have going on, well-done lad**. 
   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -63,21 +73,27 @@ <h2 id="sub-main-header">I am a Software Developer</h2>
...
-      <p>A daily selection of privately personalized reads; no accounts or sign-ups 
-        required. has been the industry's standard dummy text ever since the 1500s, 
-        when an unknown printer took a standard dummy text.</p>
-        <ul class="flex-row">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-        <button type="button">See Project</button>
+      <div id="multipost-text" class="flex-column">
```
Comment (mke2111) [inline index.html:84]: - [ ] Adjust a little the padding/margin on the left side to equal that on the right side.
Screenshot from 2021-08-20 16-02-24

File: style.css [hunk 2]
```diff
@@ -345,3 +357,221 @@ footer {
   border-radius: 10px;
   margin: 20px 0;
 }
+
+@media only screen and (min-width: 1024px) {
```
Comment (mke2111) [inline style.css:361]: - [ ] The desktop version looks great. However, you have designed the mobile version specifically for one screen size (376px), for this reason, it is not responsive.
Screenshot from 2021-08-20 16-06-26

**Review by AshakaE — APPROVED** [review]
### Hi @nachosala89  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**